src/pages/DelegatoryValidator/WalletConnectionDialog.tsx

```diff
@@ -29,10 +29,8 @@ export default function WalletConnectionDialog({
       <Box sx={{display: "flex", justifyContent: "center", marginY: 2}}>
         <ConnectWalletModalIcon />
       </Box>
-      <DialogTitle>
-        <Typography variant="h5" textAlign="center">
-          Please connect your wallet
-        </Typography>
+      <DialogTitle textAlign="center">
```

Comment:
do we also want to have `variant="h5"` here?

Reply:
this DialogTitle includes two texts, not both of them need h5 size here.